dev/bootstrap_dev.sh

```diff
@@ -240,42 +249,40 @@ if [[ "$CONFIRMED_GITHUB_REPOSITORY_OWNER" != "none" ]]; then
 
   # This is a bootstrap pull of the development scripts.
   if [[ ! -e "spinnaker" ]]; then
+    existing_spinnaker=false
     git_clone $CONFIRMED_GITHUB_REPOSITORY_OWNER "spinnaker" "spinnaker"
   else
+    existing_spinnaker=true
     echo "spinnaker/ already exists. Don't clone it."
   fi
 
-  # Pull the spinnaker source into a fresh build directory.
-  ./spinnaker/dev/refresh_source.sh --pull_origin \
-      --github_user $CONFIRMED_GITHUB_REPOSITORY_OWNER
-fi
-
-# Prepare spinnaker-local.yml
-if [[ ! -f $HOME/.spinnaker/spinnaker-local.yml ]]; then
-    # This has a side effect in which it will produce a spinnaker-local if
-    # it did not already exist. Since there is no "bogus" microservice, this
-    # will not actually do anything other than produce the spinnaker-local.
-    # The default spinnaker-local will be configured for this machine as
-    # a better starting point than otherwise.
-    ./spinnaker/dev/stop_dev.sh bogus >& /dev/null || true
-fi
-
-# Some dependencies of Deck rely on Bower to manage their dependencies. Bower
-# annoyingly prompts the user to collect some stats, so this disables that.
-echo "{\"interactive\":false}" > ~/.bowerrc
+  echo "Setting up buildtool virtual environment in $PWD/venv"
+  virtualenv venv
```

Comment:
This is assuming `virtualenv` and `pip`, would we want to ensure these here since this is a bootstrap script?

Reply:
This runs after the install_development script which installs them.

Reply:
Can you add that assumption in a comment here?